[2021-02-15 10:38:50,642] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: NDBC_51001.clean_and_load 2021-02-15T05:00:00+00:00 [queued]>
[2021-02-15 10:38:50,651] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: NDBC_51001.clean_and_load 2021-02-15T05:00:00+00:00 [queued]>
[2021-02-15 10:38:50,651] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-15 10:38:50,651] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-15 10:38:50,651] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-15 10:38:50,659] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): clean_and_load> on 2021-02-15T05:00:00+00:00
[2021-02-15 10:38:50,661] {standard_task_runner.py:52} INFO - Started process 20526 to run task
[2021-02-15 10:38:50,668] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'NDBC_51001', 'clean_and_load', '2021-02-15T05:00:00+00:00', '--job-id', '98', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ndbc_dag.py', '--cfg-path', '/var/folders/y8/ykx6qdr96dldyw_z287s5l580000gn/T/tmpe_lpz6fa', '--error-file', '/var/folders/y8/ykx6qdr96dldyw_z287s5l580000gn/T/tmpvox7jx6z']
[2021-02-15 10:38:50,670] {standard_task_runner.py:77} INFO - Job 98: Subtask clean_and_load
[2021-02-15 10:38:50,711] {logging_mixin.py:104} INFO - Running <TaskInstance: NDBC_51001.clean_and_load 2021-02-15T05:00:00+00:00 [running]> on host chriss-mbp.lan1
[2021-02-15 10:38:50,735] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=holloway.christopher.t@gmail.com
AIRFLOW_CTX_DAG_OWNER=chris
AIRFLOW_CTX_DAG_ID=NDBC_51001
AIRFLOW_CTX_TASK_ID=clean_and_load
AIRFLOW_CTX_EXECUTION_DATE=2021-02-15T05:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-15T05:00:00+00:00
[2021-02-15 10:38:51,106] {base.py:65} INFO - Using connection to: id: NDBC_51001. Host: localhost, Port: 5400, Schema: NDBC_51001DB, Login: chrisholloway, Password: None, extra: None
[2021-02-15 10:38:51,113] {dbapi.py:180} INFO - Running statement: INSERT INTO Detailed_Wave_Data_Table (significant_wave_height, swell_height, swell_period,
	 wind_wave_height, wind_wave_period, average_period) VALUES (%s, %s, %s, %s, %s, %s);, parameters: (2.5, 2.5, 11.4, 0.3, 4.0, 8.8)
[2021-02-15 10:38:51,115] {dbapi.py:186} INFO - Rows affected: 1
[2021-02-15 10:38:51,117] {python.py:118} INFO - Done. Returned value was: None
[2021-02-15 10:38:51,121] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=NDBC_51001, task_id=clean_and_load, execution_date=20210215T050000, start_date=20210215T203850, end_date=20210215T203851
[2021-02-15 10:38:51,133] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-02-15 10:38:51,142] {local_task_job.py:146} INFO - Task exited with return code 0
